hw/top_earlgrey/dv/chip_sim_cfg.hjson

```diff
@@ -227,6 +227,7 @@
       uvm_test_seq: chip_sw_lc_ctrl_transition_vseq
       sw_images: ["sw/device/tests/lc_ctrl_transition_test:1"]
       en_run_modes: ["sw_test_mode"]
+      reseed: 10
```

Comment:
With 10 reseed, kmac_data and test_exit_token toggle coverage are 100%:
![image](https://user-images.githubusercontent.com/11466553/133864642-16765c75-0217-4b9a-9f49-f405583eba7e.png)
![image](https://user-images.githubusercontent.com/11466553/133864682-ceb8ebec-9a78-4cd0-a7a5-01494ec72933.png)
Next step will be supporting other tokens' toggle coverage.

Reply:
Super awesome!! This will serve as a great reference test for all other blocks. Thank you very much for working this out.

Reply:
But really though - were 10 seeds needed? Since we are iterating within test by issuing reset between interations, I would assume we could achieve this goal with just one seed.

Reply:
Yeah, we randomize the num_trans as below:
```
  constraint num_trans_c {
    num_trans inside {[1:5]};
  }
```
Let me try a larger rand number, but I was worried this test would run for too long (say around 30 min). I thought might be better to break into smaller tests?

Reply:
Ohh, right, this count is also randomized. Let's set the lower bound to 2 instead of 1? 

We want to hit 100% coverage on those signals with the minimal per-test runtime (5-10 mins is ok), and with the smallest `reseed`. I asked the question because I was not sure if reseed was set to 10 arbitrarily (and I felt we could reduce it further). Since we are only covering toggles on those IOs, our state space is much smaller.

Reply:
Sounds good! Thanks for elaborating. Let me play around with these parameters and limit the case to < 10 min with the smallest reseed :)

Reply:
Hi Sri, 
I did some experiment, and  9 reseed actually is the minimal that meets the requirements. 
Just a few stats:
1). KMAC toggle has 382*2 = 764 bits, we need to cover the toggle coverage for all these bits.
2). With constraints {2:5}, the max run time is 25 min in cloud (I think it happens when the num_trans = 5).
     I checked the timing in detail, when num_trans = 5, there is totally 10 resets.
     I ran a "chip_aes_smoketest", the run time is around 5 min in cloud.  I think the main runtime comes from resetting and getting the chip ready.

Reply:
Sounds good, thanks for running this experiment. Lets set reseed to 15 and limit num_trans to 2:3. WDYT?

Reply:
Thanks Sri, with the setting you proposed, the toggle coverage is still 100% and total max run time is 13 min.